src/webots/gui/WbGuiApplication.cpp

```diff
@@ -482,13 +490,20 @@ WbSimulationState::Mode WbGuiApplication::startupModeFromPreferences() const {
 
   if (startupMode == "Real-time")
     return WbSimulationState::REALTIME;
-  if (startupMode == "Run")
-    return WbSimulationState::RUN;
   if (startupMode == "Fast")
     return WbSimulationState::FAST;
   return WbSimulationState::PAUSE;
 }
 
+bool WbGuiApplication::show3DViewFromPreferences() const {
+  WbPreferences *const prefs = WbPreferences::instance();
+  const QString show3DView(prefs->value("General/show3DView").toString());
+
+  if (show3DView == "false")
+    return false;
+  return true;
```

Comment:
It seems this could be simplified
```suggestion
  return show3DView != "false"
```